ReactAndroid/src/main/java/com/facebook/react/animated/InterpolationAnimatedNode.java

```diff
@@ -105,13 +112,51 @@ private static int findRangeIndex(double value, double[] ranges) {
 
   private final double mInputRange[];
   private final double mOutputRange[];
+  private String mPattern;
+  private double mOutputs[][];
+  private final boolean mHasStringOutput;
+  private final Matcher mSOutputMatcher;
   private final String mExtrapolateLeft;
   private final String mExtrapolateRight;
   private @Nullable ValueAnimatedNode mParent;
+  private boolean shouldRound;
```

Comment:
`mShouldRound`?

Reply:
Ah, good catch

Reply:
Resolved in https://github.com/msand/react-native/commit/1e984241ef926c83cdf590a6384c72ac58afc37d